libbeat/processors/actions/drop_sampling.go

```diff
@@ -74,13 +74,15 @@ func (d *dropSampling) Run(b *beat.Event) (*beat.Event, error) {
 	}
 	d.annotated.Inc()
 
-	if id, ok := collapseSectionIdInEvent(b); ok {
+	if id, hasId := collapseSectionIdInEvent(b); hasId {
```

Comment:
(nitpick) it needs to be `ID` according to [Go Code Reviews Comments](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).